src/components/BalanceComponent.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <CardComponent
-      :header="$t('c_currentSaldo.saldo')"
-      :action="showOption ? 'Increase Online' : ''"
+      :header="$t('c_currentBalance.balance')"
```

Comment:
If you're importing t from vue-i18n, you can leave out the $ right? It doesn't need to use the global definition then.